docs/configuration.rst

```diff
@@ -290,9 +292,11 @@ The following environment variables for the tracer are supported:
 
    DD_TRACE_HEADER_TAGS:
      description: |
-         A map of case-insensitive header keys to tag names. Automatically applies matching header values as tags on root spans.
+         A map of case-insensitive header keys to tag names. Automatically applies matching header values as tags on root spans. For example if
+         ``DD_TRACE_HEADER_TAGS=User-Agent:http.useragent,content-type:http.content_type``. The value of the header will set in ``http.useragent`` and ``http.content_type`` tags.
 
-         For example, ``User-Agent:http.useragent,content-type:http.content_type``.
+        If a tag value is not supplied the header will be normalized and set. For example if
+         ``DD_TRACE_HEADER_TAGS=User-Agent,content-type``. The value of the header will set in ``user-agent`` and ``content-type`` tags.
```

Comment:
Input like `DD_TRACE_HEADER_TAGS=User-Agent,content-type` should result in span tags prefixed with `http.request.headers.`, i.e, http.request.headers.user-agent and http.request.headers.content-type.

See first couple examples under "Examples" - https://datadoghq.atlassian.net/wiki/spaces/APMINT/pages/2302444638/DD+TRACE+HEADER+TAGS#Examples
Note that that^ is how the Go tracer handles input with no "mapping" value, and that's how the system test is configured: https://github.com/DataDog/system-tests/blob/main/tests/test_library_conf.py#L31